cmd/openshift-install/create.go

```diff
@@ -186,6 +193,49 @@ func runTargetCmd(targets ...asset.WritableAsset) func(cmd *cobra.Command, args
 	}
 }
 
+// appendRouterCAToClusterCA appends router CA to cluster CA in kubeconfig
+func appendRouterCAToClusterCA(config *rest.Config, directory string) (err error) {
+	client, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return errors.Wrap(err, "creating a Kubernetes client")
+	}
+
+	routerSecret, err := client.CoreV1().Secrets("openshift-ingress-operator").Get("router-ca", metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		return err
+	}
+	routerCrtBytes := routerSecret.Data["tls.crt"]
+	absDir, err := filepath.Abs(directory)
```

Comment:
Why do we need to use the absolute directory?

Reply:
no we do not, fixed